app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -397,7 +395,7 @@ public void checkObjectives(Location location) {
         }
 
         if(!addedCoins){
-            //placeRandomCoins(1, 100000);
+            placeRandomCoins(2, 6);
```

Comment:
- A small comment saying that we are placing 2 random coins in a radius of 6 meters can be useful for clarity.
- Also why only 2 coins?
- And adding a constant for setting coins and radius is maybe even better than a comment.

Reply:
That is done, you are looking at an outdated version of the code

Reply:
sorry, my bad